layer-atlas/src/main/java/com/layer/atlas/util/picasso/PicassoLayer.java

```diff
@@ -0,0 +1,35 @@
+package com.layer.atlas.util.picasso;
+
+import android.content.Context;
+
+import com.layer.atlas.util.picasso.requesthandlers.MessagePartRequestHandler;
+import com.layer.sdk.LayerClient;
+import com.squareup.picasso.Picasso;
+
+public class PicassoLayer {
```

Comment:
Remove this, pass Picasso instance in initMethod, just like with LayerClient, we don't need 2 different instances of Picasso.